tutorials/en_us/Configuring-Your-Server.md

```diff
@@ -0,0 +1,268 @@
+
+### `generator-settings`
+
+The settings used to customise world generation
+
+Follow the format found [here](https://minecraft.gamepedia.com/Java_Edition_level_format#generatorOptions_tag_format)
+
+### `level-name`
+
+The name of your world folder in your root directory
+
+### `motd`
+
+The motd, or Message Of The Day is the message that displays under the name of the server in a client's server list.
+
+By default, all text will be white. This colour can change through the use of colour codes in the format of `\u00A7` and then one of the sixteen colour codes and six formatting codes, or using the chart found below (the column titled 'MOTD code')
+
+![Image of colour codes](https://lazerstudiozgaming.files.wordpress.com/2015/01/screen-shot-2015-01-03-at-11-27-44-am.png)
+
+### `query.port`
+
+The port for the query server (see [`enable-query`](#enable-query))
+
+### `pvp`
+
+Whether or not players can harm eachother.
+
+### `generate-structures`
+
+Whether or not to generate structures like Villages, Strongholds, Outposts, and others
+
+### `difficulty`
+
+The difficulty of the server
+
+Possible values: peaceful, easy, normal, hard
+
+### `network-compression-threshold`
+
+By default it allows packets that are n-1 bytes big to go normally, but a packet of n bytes or more gets compressed down. So, a lower number means more compression but compressing small amounts of bytes might actually end up with a larger result than what went in.
+
+Note: The Ethernet spec requires that packets less than 64 bytes become padded to 64 bytes. Thus, setting a value lower than 64 may not be beneficial. It is also not recommended to exceed the MTU, typically 1500 bytes. 
+
+Possible values: Integer, -1 means disable compression entirely, 0 means compress everything
+
+Default: 256
+
+### `max-tick-time`
+
+The max amount of milliseconds per tick before the server stops itself, assuming a crash has happened.
+
+Possible Values: Integer 0 - 2⁶³-1
+
+### `use-native-transport`
+
+Linux server performance improvements: optimized packet sending/receiving on Linux
+
+
+### `max-players`
+
+Maximum players on at one time
+
+### `online-mode`
+
+When enabled, the server will assume that it has an intenet connection and check all players joining for a valid Minecraft account.
+
+### `enable-status`
+
+Whether or not the server appears as 'online' in the server list
+
+### `allow-flight`
+
+When set to true, players who would normally be kicked for flying automatically will not be.
+
+### `broadcast-rcon-to-ops`
+
+Send rcon(see [`enable-rcon`](#enable-rcon)) console command outputs to all online operators.
+
+### `view-distance`
+
+The amount of world data the server sends the client.  Value is measured in chunks
+
+Possible Values: Integer (3-32) 
+
+### `max-build-height`
+
+The maximum build height that someone can build.
+
+Integer 0 - 256 (**must be multiple of 8**)
+(as of 1.16.5)
+
+### `server-ip`
+
+Set if you want the server to a particular IP.
```

Comment:
Maybe explain that a bit more. I don't have an idea of what it does. Only accept traffic which is going to said ip?